README.md

```diff
@@ -36,7 +36,7 @@ This should implement the following:
 
 (Windows setup is not implemented in `make` yet. So we will have to install the required files manually)
 
-1. Install Arduino IDE (v1.8.5) from https://www.arduino.cc/en/Main/OldSoftwareReleases#previous
+1. Install Arduino IDE (v1.8.10) from https://www.arduino.cc/en/Main/Software
```

Comment:
this can probably be a markdown link. Also is this link gonna expire eventually? can we get a permalink to the latest version here?

Reply:
I forgot to address this. I'll check if there's a latest version permalink available and update the docs once all the firmware changes are in (since I will have to update docs for that anyway)